panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,25 +50,34 @@
         </app-toolbar>
       </app-header>
 
+      <app-localstorage-document
+        key='panel-dev-service-state-domain'
+        data='{{domain}}'>
+      </app-localstorage-document>
+      <app-localstorage-document
+        key='panel-dev-service-state-service'
+        data='{{service}}'>
+      </app-localstorage-document>
+      <app-localstorage-document
+        key='panel-dev-service-state-servicedata'
+        data='{{serviceData}}'>
+      </app-localstorage-document>
+
       <div class$='[[computeFormClasses(narrow)]]'>
```

Comment:
The method computeFormClasses can also be removed. Instead just have `content`. I am not 100% sure why we needed `fit` anymore. Make sure the background remains all white. If that's not the case you should add `fit` back and also `iron-positioning`.